wger/core/views/user.py

```diff
@@ -529,3 +529,39 @@ def get_context_data(self, **kwargs):
                                           _('Gym')],
                                  'users': context['object_list']['members']}
         return context
+
+
+class InactiveUserListView(LoginRequiredMixin, PermissionRequiredMixin, ListView):
+    '''
+    Overview of inactive users in the instance
+    '''
+    model = User
+    permission_required = ('gym.manage_gyms',)
+    template_name = 'user/inactive_list.html'
+
+    def get_queryset(self):
+        '''
+        Return a list with the users, not really a queryset.
+        '''
+        out = {'admins': [],
+               'members': []}
+
+        for u in User.objects.select_related('usercache', 'userprofile__gym').all():
```

Comment:
I have replaced the loop with a comprehension. The admins key was meant for future extension where there would be a separation of ordinary users and people with admin privileges. For the moment it is unnecessary so I have removed it